doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -235,33 +245,39 @@ The static input files for global configurations are listed and described in :nu
    * - Filename
      - Description
      - Used in resolution
-   * - runoff.daitren.clim.1440x1080.v20180328.nc
+   * - runoff.daitren.clim.1440x1080.v20180328.nc 
```

Comment:
It's probably worth adding a note before this table that not all files are required for all resolutions. Like, this file is only required when running 0.25 resolution.